editor/assets/stylesheets/_variables.scss

```diff
@@ -19,6 +19,7 @@ $light-gray-300: #edeff0;
 $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
 $white: #fff;
+$red: #dc3232;
```

Comment:
The color was introduced in #924 and may have been ported directly from Calypso. I think it'd be fair to update `$alert-red` to the core standard.